src/game.py

```diff
@@ -0,0 +1,57 @@
+from board import Board
+from os import system
+
+
+class Game:
+    def __init__(self):
+        self.board = Board()
+
+    def start(self):
+        system('clear')
+        self.display_title()
+        while True:
+            system('clear')
+            self.display_board()
+            self.display_instructions()
+            input_command = ''
+            while (input_command.lower() not in [Board.EAST, Board.WEST, Board.NORTH, Board.SOUTH]):
+                input_command = input('Use ' + Board.NORTH + ', ' + Board.SOUTH +
+                                      ', ' + Board.EAST + ', ' + Board.WEST + ' to tilt the board: ')
+
+            self.board.tilt(input_command)
+
+    def display_board(self):
```

Comment:
You should unit test this method at least once

Reply:
You mean unit test the display of the board?

Reply:
yes. That will force you to separate string formatting and string output.

Reply:
done